src/components/spreadsheet/spreadsheet.ts

```diff
@@ -181,6 +182,7 @@ export class Spreadsheet extends Component<SpreadsheetProps, SpreadsheetChildEnv
       toggleSidePanel: this.toggleSidePanel.bind(this),
       _t: Spreadsheet._t,
       clipboard: navigator.clipboard,
+      FocusableElement: new FocusableElement(),
```

Comment:
capitalised ? it looks like a class while it's actually an instance
```suggestion
      focusableElement: new FocusableElement(),
```

Reply:
sadge find & replace